src/types.ts

```diff
@@ -200,3 +200,14 @@ export const OnboardingEarn = {
   moneyLaundering: 500,
   tweet: 1000,
 }
+
+// TODO: Add types for payer, payee and metadata
+export type IAddTransactionOnUsPayment = {
+  description: string
+  sats: number
+  metadata: Record<string, unknown>
+  payerUser: any
```

Comment:
use of `any` is no longer allowed with our eslint/typescript rules

Reply:
`any` does give a warning (not an error), so I'm using our `typeof User` workaround for now. I don't know if there's a better way to do it (without adding types to our mongoose code).
@samerbuna Do you have a suggestion on this?

Reply:
I don't think `typeof User` is a good idea but I see it used in the mongoose schema too, so maybe leave it for now and create a new issue to refactor all mongoose types to use interfaces extending mongoose documents.